test/unit/message-service.test.js

```diff
@@ -0,0 +1,105 @@
+const mockConfig = {
```

Comment:
Supporting both SQS and AMQP queues in the microservice has added complexity to the project.

Rather than reference both queue technologies in the same service it would be cleaner to separate them into their own services, i.e. amqp-message-service and sqs-message-service. 

This would reduce the number of test stubs required in the SQS message service test as the AMQP service is no longer referenced.